File: cpp/math/main_10250.cpp
#include <iostream>

using namespace std;

int main() {
    int count;
    cin >> count;
    for (int i = 0; i < count; ++i) {
	    int h, w, n;
        cin >> h >> w >> n;
        int floor = n % h;
        int room = n / h + 1;
        if (floor == 0) {
            floor = h;
            room -= 1;
        }
        cout << floor;
        if (room < 10) 
            cout << 0;
        cout << room << endl;
    }
}

File: kotlin/step/math/main_10250.kt
fun main() {
    val lines = readLine()!!.split("\n")
    for (line in lines.subList(1, lines.lastIndex + 1)){
        val (h, _, n) = line.split(" ").map { it.toInt() }
        var floor = n % h
        var room = n / h + 1
        if (floor == 0) {
            floor = h
            room -= 1
        }
        println("$floor${if (room < 10) "0" else ""}${room}")
    }
}